### ISSUE TITLE
OpenApi (swagger) UI

### ISSUE THREAD

**Issue by fbaldo31** [alosaur/alosaur#137]
Thank you for sharing this really promising lib !
Do you plan to add serving swagger-ui with the openapi module ?

I am working on this, so i will be able to can provide a PR in next days if you want.

**Comment by irustm** [comment]
Thank you! Yes, it would be nice to have the swagger ui of openapi. Feel free to create a PR

**Comment by fbaldo31** [comment]
Hi,
I have a first result, but i face to an issue with `AlosaurOpenApiBuilder.addServer` method.
When i use it like this : 
`.addServer({ url: 'http://localhost:8000', description: 'Local server'})`

The produced url when i test a route with `Try it out` is like `http://localhost:8000/api/http://localhost:8000/app/xxx`
Do you have an idea why we get this behaviour ?

**Comment by irustm** [comment]
can you send me your compiled openapi.json?

**Comment by fbaldo31** [comment]
I don't know what happend but the issue seems to be solved now :upside_down_face: 

What do you think about swagger-ui files ?
Do we only manage only the case of an app created with the cli ?
In this case, i imagine the cli creating a swagger-ui folder, but you may have another idea ?

**Comment by irustm** [comment]
Yes, I think the swagger-ui files should be somewhere near openapi.
